src/public/modules/forms/admin/views/list-forms.client.view.html

```diff
@@ -5,7 +5,9 @@
   <div id="list-tab">
     <section data-ng-init="vm.listForms()">
       <div id="list-form" class="container" ng-if="vm.user">
-        <react-switch-banner-component></react-switch-banner-component>
+        <div id="react-switch-banner" ng-if="vm.reactAdminRollout > 0">
+          <react-switch-banner-component></react-switch-banner-component>
```

Comment:
my angularjs is rusty, but i think we can put the `ng-if` directly on the component
```tsx
<react-switch-banner-component ng-if="vm.reactAdminRollout > 0"></react-switch-banner-component>
```

Reply:
just tried it, can it works!